Sources/Sentry/Public/SentrySDK.h

```diff
@@ -6,11 +6,18 @@
 
 @protocol SentrySpan;
 
-@class SentryOptions, SentryEvent, SentryBreadcrumb, SentryScope, SentryUser, SentryId,
-    SentryUserFeedback, SentryTransactionContext;
+@class SentryBreadcrumb;
+@class SentryEvent;
+@class SentryFeedback;
+@class SentryId;
```

Comment:
Thanks 🥇

Reply:
heh i didn't even mean for this to go in here, i think.